src/test/java/org/jenkinsci/plugins/workflow/steps/ArtifactArchiverStepTest.java

```diff
@@ -1,22 +1,26 @@
 package org.jenkinsci.plugins.workflow.steps;
 
 import java.util.Arrays;
+import jenkins.model.ArtifactManagerConfiguration;
 import jenkins.util.VirtualFile;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.plugins.workflow.DirectArtifactManagerFactory;
 import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
 import org.jenkinsci.plugins.workflow.job.WorkflowJob;
 import org.jenkinsci.plugins.workflow.job.WorkflowRun;
-import org.junit.Assert;
+import static org.junit.Assert.*;
+import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
+import org.jvnet.hudson.test.BuildWatcher;
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.JenkinsRule;
 
-/**
- * @author Kohsuke Kawaguchi
- */
-public class ArtifactArchiverStepTest extends Assert {
+public class ArtifactArchiverStepTest {
+
+    @ClassRule public static BuildWatcher watcher = new BuildWatcher();
```

Comment:
What is this used for?

Reply:
Just so that the test log includes the log from the build run at line 90.